code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -12,6 +12,9 @@
 	var/obj/item/weapon/disk/nuclear/the_disk = null
 	var/active = 0
 
+	Destroy()
+		active = 0
+		return ..()

```

Comment:
This should really null out its reference to `the_disk`, too. Also, the pinpointer code has a bunch of spots that should, but don't, check whether references are GC'd, so they leave dangling references to stuff they're pointing to.